misc/maintenance/fix_cpplint_whitespace.sh

```diff
@@ -3,11 +3,11 @@
 
 # Let's run a set of in-place modifications by sed-commands,
 for file in $@; do
-  sed -i 's/; \/\//;  \/\//' $file # '; //' -> ';  //'
-  sed -i 's/{ \/\//{  \/\//' $file # '{ //' -> '{  //'
-  sed -i 's/} \/\//}  \/\//' $file # '} //' -> '}  //'
-  sed -i 's/for(/for (/' $file     # 'for(' -> 'for ('
-  sed -i 's/if(/if (/' $file       # 'if(' -> 'if ('
-  sed -i 's/\s\s*$//' $file        # 'remove white-space at the end of lines'
+  perl -i -pe 's/; \/\//;  \/\//' $file # '; //' -> ';  //'
+  perl -i -pe 's/{ \/\//{  \/\//' $file # '{ //' -> '{  //'
+  perl -i -pe 's/} \/\//}  \/\//' $file # '} //' -> '}  //'
+  perl -i -pe 's/for(/for (/' $file     # 'for(' -> 'for ('
+  perl -i -pe 's/if(/if (/' $file       # 'if(' -> 'if ('
+  perl -i -pe 's/\s\s*$//' $file        # 'remove white-space at the end of lines'
```

Comment:
I tested this and it seemed to  remove the final newline.

Reply:
@danpovey, as a project to use clang-tidy is under way, maybe we should not touch this file at all? It's a maintainer's file, so presumably sed is fine here for the time being, only a handful of people run it if ever.